### DISCUSSION TITLE
Any announcements about VisiData 3.0?

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#2332] [Q&A]
I have not seen any toots or tweets from the VisiData accounts saying that 3.0.X is available.  Or about the request for help for writing guides.  It feels like it would be helpful to people who are fans, but do not follow too closely. 

 Or is the plan to wait until all the guides exist?

I feel like @saulpw  and @anjakefala  you put in a lot of fantastic work to get 3.0 out.  Plus at lot of help from @midichef and others.

**Comment by saulpw** [comment]
Yeah I'm not so great at the marketing!  I'm no longer on Twitter, but I did post on Mastodon about VisiData.30 a couple of months ago: https://fosstodon.org/deck/@saulpw/111718896695101720

We're busy working on another project (to be announced in a few days).  I only wish there was someone who loved VisiData who would be willing to post more regularly on social media on our behalf.

**Comment by anjakefala** [comment]
> Or is the plan to wait until all the guides exist?

No, I would love it if folks could get the word out! I posted on the discord, and here because that is where superusers and regular contributors mostly hang out. I definitely want help for contributing to guides, that was the reasoning behind investing in that blogpost. If anyone feels blocked on contributing, please talk to me! I'm putting more of my spare time into $SecretProject for the next bit. =)

**Reply by frosencrantz** [reply]
Excited to hear about $SecretProject.

I was thinking that it would be helpful to send notice of the new release and blog posts to the Mastodon and Twitter feeds for VisiData.  I currently do not have time or I might be willing to volunteer to help.  I am happy to see new people participating in the VisiData community, and hope they might help get the message out.
